drivers/hmis/app/graphql/mutations/add_household_members_to_enrollment.rb

```diff
@@ -0,0 +1,65 @@
+module Mutations
+  class AddHouseholdMembersToEnrollment < BaseMutation
+    argument :household_id, ID, required: true
+    date_string_argument :start_date, 'Start date with format yyyy-mm-dd', required: true
+    argument :household_members, [Types::HmisSchema::EnrollmentHouseholdMemberInput], required: true
+
+    field :enrollments, [Types::HmisSchema::Enrollment], null: true
+    field :errors, [Types::HmisSchema::ValidationError], null: false
+
+    def validate_input(household_id:, start_date:, household_members:)
+      errors = []
+      errors << InputValidationError.new('Entry date cannot be in the future', attribute: 'start_date') if Date.parse(start_date) > Date.today
+
+      has_hoh_enrollment = Hmis::Hud::Enrollment.viewable_by(current_user).exists?(household_id: household_id, relationship_to_ho_h: 1)
+      errors << InputValidationError.new("Cannot find Enrollment for household with id '#{household_id}'", attribute: 'household_id') unless has_hoh_enrollment
+      errors << InputValidationError.new('Enrollment already has a head of household designated', attribute: 'household_members') if has_hoh_enrollment && household_members.find { |hm| hm.relationship_to_ho_h == 1 }
```

Comment:
I think we need to be forgiving in the case where an household doesn't have an HoH. We'll have data imported from other sources initially, so it could happen. Correct me if I'm wrong @eanders. So- I suggest removing line 15 and keeping line 16.

Reply:
Actually, maybe we assign a HoH randomly on import. Do we do that @eanders ?

Reply:
We definitely see data where we are missing an HoH and need to support that, though it should be in an error state that requires fixing before changes can be made.

We do have a process for setting the HoH and we'll probably invoke it on import for any data we plan to use in the HMIS.  (It's not random, but the idea is the same.)  I'm ok assuming every household will have an HoH if that makes this cleaner.

Reply:
Yes agreed it should be rendered to the user as an error state, but unless we remove line 15, they _can't_ fix it via the graphql API.

Reply:
can we see what happens when you have an enrollment/household with no HoH?

Reply:
> can we see what happens when you have an enrollment/household with no HoH?

It will raise an error saying `Cannot find Enrollment for household with id X` . Unless I'm totally misreading the code.


> What do you all want to do here?

My suggestion: only raise the error on line 15 ("Cannot find Enrollment for household with id..") if the `household_id` doesn't exist.

Reply:
@gigxz Yeah, that's correct. And agreed that I think that's the best solution and probably more correct

Reply:
I went ahead and made the change to use `.find_by(household_id: household_id)` for the household existence check and pulling the project ID, since I think that should work fine and it's more flexible. I did keep the check for `RelationshipToHoH` in the validation to ensure we're not adding a second head of household though.